src/engines/html5/html5.js

```diff
@@ -240,7 +252,35 @@ export default class Html5 extends FakeEventTarget implements IEngine {
   get id(): string {
     return Html5.id;
   }
-
+  /**
+   * Listen to the video element events and triggers them from the engine.
+   * @public
+   * @param {boolean} stopSeeking don't seek to the last detach point
+   * @returns {void}
+   */
+  attachMediaSource(stopSeeking: ?boolean): void {
+    if (!this._mediaSourceAdapter) {
+      this.reset();
+      this._init(this._source, this._config);
+      if (!isNaN(this._lastTimeDetach) && !stopSeeking) {
+        this._eventManager.listenOnce(this, Html5EventType.CAN_PLAY, () => {
+          this.currentTime = this._lastTimeDetach;
```

Comment:
what's about the selection tracks (audio/text), playback-rate, etc. did you verify they remain as before?

Reply:
audio / text sure, playback rate will not save..
hls.js and shaka should find the best playback rate again

Reply:
playback-rate means the speed of playback
this is must be handled by you